src/filetypes.jl

```diff
@@ -1,3 +1,34 @@
+"""
+    readXYZ(io::IO) -> Vector{AtomPosition{3}}
+
+Reads an XYZ file into a `Vector{AtomPosition{3}}`.
+"""
+function readXYZ(io::IO)
+    # Loop through each line in the file
+    itr = eachline(io)
+    # Get the number of atoms from the first line
+    n_at = parse(Int, iterate(itr)[1])
+    v = Vector{AtomPosition{3}}(undef, n_at)
+    # Skip the second line
+    iterate(itr)
+    # Loop through the rest
+    for (n, ln) in enumerate(itr)
```

Comment:
I'd maybe just have a block comment here explaining how each line should look, which will give context to reading the body of the loop here